docs/dev/libs/cep.md

```diff
@@ -778,15 +778,15 @@ next.within(Time.seconds(10))
 
 You can apply the same contiguity condition as discussed in the previous [section](#combining-patterns) within a looping pattern.
 The contiguity will be applied between elements accepted into such a pattern.
-To illustrate the above with an example, a pattern sequence `"a b+ c"` (`"a"` followed by one or more `"b"`'s followed by a `"c"`) with
+To illustrate the above with an example, a pattern sequence `"a b+ c"` (`"a"` followed by any(non-deterministic relaxed) one or more `"b"`'s followed by a `"c"`) with
```

Comment:
There's something wrong in `followed by any(non-deterministic relaxed) one or more`, i guess `one or more` should be removed?

Reply:
Actually it is not. I am not sure how to properly express contiguity between `a` and `b+` in a "spoken" language. 
What do you think about sentence like this:
> followed by any sequence of one or more `b`s
Is it more understandable?

Reply:
yes